packages/clay-progress-bar/src/ClayProgressBar.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+import classNames from 'classnames';
+
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Flag to indicate if `children` should be wrapped as `feedback`.
+	 */
+	feedback?: boolean;
+
+	/**
+	 * Determines what the color the progress bar is.
+	 */
+	status?: 'success' | 'warning';
+
+	/**
+	 * The current value of the progress bar. Should range from 0 to 100.
+	 */
+	value: number;
+}
+
+const ClayProgressBar: React.FunctionComponent<Props> = ({
+	children,
+	className,
+	feedback,
+	status,
+	value,
```

Comment:
We can give you a warning (the same as we did for ClayIcon) to ensure that are passing a value in the range of 0 to 100. What do you think?

Reply:
👍